pulsar-client/src/main/java/org/apache/pulsar/client/api/ClientBuilder.java

```diff
@@ -246,14 +246,24 @@ ClientBuilder authentication(String authPluginClassName, Map<String, String> aut
     ClientBuilder statsInterval(long statsInterval, TimeUnit unit);
 
     /**
-     * Number of concurrent lookup-requests allowed on each broker-connection to prevent overload on broker.
+     * Number of concurrent lookup-requests allowed to send on each broker-connection to prevent overload on broker.
      * <i>(default: 5000)</i> It should be configured with higher value only in case of it requires to produce/subscribe
      * on thousands of topic using created {@link PulsarClient}
      *
      * @param maxConcurrentLookupRequests
      */
     ClientBuilder maxConcurrentLookupRequests(int maxConcurrentLookupRequests);
 
+    /**
+     * Number of max lookup-requests allowed on each broker-connection to prevent overload on broker.
+     * <i>(default: 20000)</i> It should not be smaller than maxConcurrentLookupRequests.
+     * Requests that inside maxConcurrentLookupRequests already send to broker, and requests beyond
+     * maxConcurrentLookupRequests and under maxLookupRequests will wait in each client cnx.
+     *
+     * @param maxLookupRequests
```

Comment:
I would encourage add `@since` annotation in the comment to indicate when the method was introduce.

Reply:
Thanks, seems this class have a Since 2.0.0 in line 30.